client/src/app/tabs/bpmn/BpmnEditor.js

```diff
@@ -264,21 +282,21 @@ export class BpmnEditor extends CachedComponent {
       });
 
       // TODO(nikku): apply default element templates to initial diagram
-      modeler.importXML(xml, (err, warnings) => {
+      modeler.importXML(xml, (error, warnings) => {
 
-        if (warnings.length) {
-          console.log('WARNINGS', warnings);
-        }
-
-        if (err) {
-          return this.handleError({
-            error: err
-          });
+        if (error) {
+          return this.handleImport(error);
```

Comment:
Should we pass warnings in case there are some, too?

Reply:
👍